Neos.Neos/Classes/Domain/Service/UserService.php

```diff
@@ -791,9 +808,12 @@ protected function createPersonalWorkspace(User $user, Account $account)
      * @param string $accountIdentifier Identifier of the user's account
      * @return void
      */
-    protected function deletePersonalWorkspace($accountIdentifier)
+    protected function deletePersonalWorkspace(User $user)
```

Comment:
It makes sense to refer to the user here instead of the Account (as discussed), but IMO we should only pass around the user *object*  where it's really required and otherwise use the user ID. In this case it's not a big difference probably but in general the id is the "common denominator"